api/swagger/swagger.yaml

```diff
@@ -18,25 +18,21 @@ consumes:
 produces:
   - application/json
 paths:
-  /hello:
+  /users:
     # binds a127 app logic to a route
-    x-swagger-router-controller: helloWorld
+    x-swagger-router-controller: userController
     get:
-      description: Returns 'Hello' to the caller
+      description: Returns json list of users
       # used as the method name of the controller
-      operationId: hello
-      parameters:
-        - name: name
-          in: query
-          description: The name of the person to whom to say hello
-          required: false
-          type: string
+      operationId: usersGet
+      parameters: []
       responses:
         "200":
           description: Success
           schema:
-            # a pointer to a definition
-            $ref: "#/definitions/HelloWorldResponse"
+            type: array
+            items:
+              $ref: "#/definitions/User"
```

Comment:
The objects being returned do not match the schema defined by `#/definitions/User`.

Here's what Swagger UI says the response should look like:
```
[
  {
    "firstName": "string",
    "lastName": "string",
    "birthMonth": 0,
    "birthDay": 0,
    "birthYear": 0,
    "email": "string"
  }
]
```

Here's an actual response:
```
[
  {
    "id": "1",
    "firstName": "Jane",
    "lastName": "Smith",
    "birthDate": "1980-01-27",
    "email": "jane.smith@example.com"
  }
]
```

You'll need to manipulate the object you get back from the database to match the format defined in Swagger.

Reply:
You can use compisition to inherit from the `User` definition. You'll want to have all of its fields in your objects, and you'll want to add an the ID. You could call your definition `UserWithId`.

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#models-with-composition